new/LIT7412QeneLaegwala.xml

```diff
@@ -0,0 +1,73 @@
+                <title xml:lang="gez" type="normalized" corresp="#t1">La-ʾǝgʷāla māwta nafsǝya taḥaddǝgā la-mǝnt ʾƎgziʾabǝḥer ʾabuhā
+                    (Qǝne of the mibazḫu-type)</title>
+                <title xml:lang="en" corresp="#t1">Why are you abandoning my soul, orphan, (you) God, her father…</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>The author of the qǝne is not indicated in the manuscript. The type of the qǝne is indicated as 
+                    <foreign xml:lang="gez-tr">Mi-bazḥu</foreign>.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Mibazhu"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+                <language ident="gez-tr">Gǝʿǝz transcribed</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2025-04-29">Created entity</change>
+            <change when="2025-04-29" who="CH">Small updates of titles for uniformity to other Bm records</change>
+            <change when="2025-05-30" who="CH">Updated abstract for more conformity to other Bm records</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="PetermannIINachtrag24"/>, fol. 7ra-b. Division into verses is indicated 
+                    with ። .</note>
+                <ab>
+                    ሚበዝሑ፡
+                    <l n="1">ለእጓለ፡ ማውተ፡ ነፍስየ፡ ተሐድጋ፡ ለምንት፡ እግ<cb n="7rb"/>ዚአብሔር፡ አቡሃ።</l> 
```

Comment:
In fact, "orphan" is እጓለ፡ ማውታ፡. If you look closely at the last letter you will see that it is indeed ታ, with the vowel marker written deficiently: the line directed to the left downwards is missing (written very thinly, disappeared or was skipped, this happens with rushed and careless hands), only serif (dot) has been made.  So, it reads indeed ማውታ፡.